app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -250,6 +250,10 @@ class ConversationViewModel @Inject constructor(
         conversationViewState = conversationViewState.copy(downloadedAssetDialogState = Hidden)
     }
 
+    fun isFileSharingEnabled(): Boolean {
+        return isFileSharingEnabledUseCase.invoke()
```

Comment:
the invoke is not suspend fun , so I think no it doesn't need to be in Coroutine scope 
please correct me if we have to put it in C scope

Reply:
Yes ! I think it could be nice to have, since at some point it seems that this is writing/reading from shared preferences. So I think we are better safe :D

Reply:
> personally I see no reason why not :thinking, i would load off the main thread

Yes I think, we are better safe, since at some point this seems to be reading/writing from shared preferences =D